prow/cmd/deck/static/prow/prow.ts

```diff
@@ -650,7 +650,13 @@ function redraw(fz: FuzzySearch): void {
         max = 2 * 3600;
     }
     drawJobHistogram(totalJob, jobHistogram, now - (12 * 3600), now, max);
-    if (rerunStatus != null) {
+    if (rerunStatus === "denied") {
+        modal.style.display = "block";
+        rerunCommand.innerHTML = `You don't have permission to rerun that job. Try asking @cjwagner.`;
```

Comment:
Thats evil 👿
Please no 🙏

Reply:
Can we also recommand contacting on-cole to help resolve for any sandwich-related disputes?

Reply:
I'll just respond to those now.
It's a sandwich.